### DISCUSSION TITLE
TP_TSP_2

### DISCUSSION THREAD

**Discussion by MIKOU-Badr** [bhic1973/ENSAF_GSEII3#4] [Q&A]
bonjour Monsieur

S'il vous plait dans la partie 2 du TP_TSP_2 est c qu'on doit créer des blocs on on doit juste utiliser les blocs fournis par le logiciel GNU-RADIO ???

merci pour votre aide.

**Comment by bhic1973** [comment]
Bonsoir
juste les blocs fournis par GnuRadio parmi lesquelles il y a possibilité de créer de nouveaux blocs utilisant Python
